playbook.yml

```diff
@@ -129,7 +133,7 @@
   - name: GET config files from DQ-config-bucket
     shell: '{{ item }}'
     with_items:
-      - aws configure set aws_access_key_id '{{user_access_key}}'
+      - aws configure set aws_access_key_id '{{aws_access_key_id}}'
       - aws configure set aws_secret_access_key '{{secret_access_key}}'
       - aws s3 cp s3://dq-config-bucket/dq-data-ingest-linux-server/ /tmp/install/ --recursive
```

Comment:
if you absolutely have to do it this way rather than IAM profiles on the instance that grant access to the bucket, please make sure you've properly removed the credentials from the box afterwards